### ISSUE TITLE
Fails on live-reload

### ISSUE THREAD

**Issue by dr-dimitru** [VeliovGroup/Meteor-Internationalization#2]
Function [`renderString`](https://github.com/VeliovGroup/Meteor-Internationalization/blob/master/ostrio:i18n.coffee#L644) does not returns values on Live-updates without timeout, but if you go by routes

**Comment by dr-dimitru** [comment]
Currently can be fixed with [meteorhacks:subs-manager package](https://github.com/meteorhacks/subs-manager)

``` coffeescript
subs = new SubsManager()

Router.configure
  waitOn: ->
    return subs.subscribe 'i18n'
```

**Comment by dr-dimitru** [comment]
fixed in `v2.0.0`
